src/styles.css

```
@@ -1,311 +1,487 @@
   align-items: center;
-  padding: 1rem;
-  background-color: var(--pixel-button-bg);
-  border: 2px solid var(--pixel-border);
-  color: var(--pixel-fg);
+  width: fit-content;
+  padding: 0.24rem 0.5rem;
+  border-radius: 999px;
+  background: rgba(255, 252, 255, 0.75);
+  border: 1px solid rgba(63, 42, 34, 0.12);
+  color: var(--retro-brown);
+  font-family: "SFMono-Regular", Menlo, Consolas, monospace;
```

Stylelint (17.3.0)

[error] Unexpected quotes around "SFMono-Regular" (font-family-name-quotes)

(font-family-name-quotes)